File: plusmall-business-web/src/main/webapp/js/service/goodsService.js
app.service('goodsService',function ($http) {

    //添加
    this.add=function (goodsGroupEntity) {
        return $http.post('../goods/add.do',goodsGroupEntity);
    }

    //查找
    this.search=function (pageNum,pageSize,searchEntity) {
        return $http.post('../goods/search.do?pageNum='+pageNum+'&pageSize='+pageSize,searchEntity);
    }
})

File: plusmall-business-interface/src/main/java/com/plusmall/business/GoodsService.java
package com.plusmall.business;

import com.plusmall.commons.PageResult;
import com.plusmall.model.TbGoods;
import com.plusmall.pojogroup.Goods;

/**
 * @Description:
 */
public interface GoodsService {
	public void add(Goods goods) throws NullPointerException;
	public PageResult search(int pageNum, int pageSize, TbGoods tbGoods);
}
